code/ui-utils/src/main/java/com/adobe/marketing/mobile/utils/UiImageConstants.kt

```diff
@@ -0,0 +1,15 @@
+package com.adobe.marketing.mobile.utils
+
+import java.util.concurrent.TimeUnit
+
+object UiImageConstants {
+    const val DEFAULT_BITMAP_QUALITY = 100
+    const val DEFAULT_DOWNLOAD_TIMEOUT_SECS = 10
+    const val CAROUSEL_MAX_BITMAP_WIDTH = 300
+    const val CAROUSEL_MAX_BITMAP_HEIGHT = 200
```

Comment:
Since these two are push template specific constants, please keep them in `PushTemplateConstants` and accept it as method parameters if needed